README.md

```diff
@@ -50,3 +51,25 @@ API Setup
 
 
 [AB2D Deploy](Deploy/README.md)
+
+## Installing and Using Pre-commit
+
```

Comment:
Since this is a compliance measure, could we add a line here along the lines of "Anyone committing to this repo must use the pre-commit hook to lower the likelihood that secrets will be exposed."